dodo.py

```diff
@@ -1,12 +1,23 @@
-# change the URLs in `./repos.yml`
+"""doit for interactive testing of accessibility in Jupyter
+"""
+import hashlib
+import http.cookiejar
+import json
 import os
 import pathlib
-import doit.tools
-import json
+import random
+import re
 import shutil
-import sys
 import subprocess
-from yaml import safe_load
+import sys
+import time
+import urllib.request
+
+import toml
+
```

Comment:
Is there a reason for the spaces between toml, jsonschema, and doit.tools imports?

Reply:
dunno, it's whatever `isort` + `black` think is best